hipe4ml/tree_handler.py

```diff
@@ -48,7 +46,8 @@ def __init__(self, file_name=None, tree_name=None, columns_names=None, **kwds):
             for file in self._files:
                 if self._tree is not None:
                     self._full_data_frame = self._full_data_frame.append(
-                        uproot.open(file)[self._tree].pandas.df(branches=columns_names, **kwds), ignore_index=True)
+                        uproot.open(f'{file}:{self._tree}').arrays(filter_name=columns_names, library='pd'),
```

Comment:
I would not remove the possibility to use kwds for uproot.open(). There are many optional features (parallel reading, max number of entries ecc..)  that can be useful for our analyses

Reply:
Done